### PR TITLE
Task Browser into Dev

### PR DESCRIPTION
**Pull request by karimkefi**
Checked Browser compatability, updated formatting and content.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta charset="UTF-8">
     <title>Karim Portfolio</title>
     <link href="style.css" rel="stylesheet" type="text/css">
+	<link href="normalize.css" rel="stylesheet" type="text/css">
```
Comment (nsantiagoblair) [inline index.html:8]: unnecessary space.
I think the proper order for link tag attributes is `rel`, `type`, `href`. I might be wrong though...

File: index.html [hunk 2]
```diff
@@ -72,22 +73,27 @@ <h1>WHAT I AM LEARNING</h1>
     <div class="headerImage2" id="aboutmeLocation">
         <section class="container">
                 <div class="articleBoxLarge articleBoxMedium articleBoxSmall">
-                    <img src="Img/london.jpg">
-                    <p>Bla bla bla Bla bla bla Bla bla bla Bla bla bla
-                        Bla bla blaBla bla blaBla bla blaBla bla bla
-                        Bla bla blaBla bla blaBla bla bla</p>
+                    <img src="Img/london.jpg" alt="St Pauls">
+                    <p>Born and bread in London. I went to a French school in London (a long story for another day)
```
Comment (nsantiagoblair) [inline index.html:77]: whitespace

File: index.html [hunk 3]
```diff
@@ -72,22 +73,27 @@ <h1>WHAT I AM LEARNING</h1>
...
+						I needed a job (any job) and found myself being interviewed in the city and getting
+						a job as a junior accountant for an investment bank where I worked until the big move.
+					</p>
                 </div>
                 <div class="articleBoxLarge articleBoxMedium articleBoxSmall">
-                    <img src="Img/bathview-KarimOwn.jpg">
-                    <p>Bla bla bla Bla bla bla Bla bla bla Bla bla bla
-                        Bla bla blaBla bla blaBla bla blaBla bla bla
-                        Bla bla blaBla bla blaBla bla bla</p>
+                    <img src="Img/bathview-KarimOwn.jpg" alt="View of Bath">
```
Comment (nsantiagoblair) [inline index.html:85]: whitespace

File: style.css [hunk 4]
```diff
@@ -195,14 +194,13 @@ aside {
 .articleBoxLarge p {
     display: inline-block;
     float: left;
-    border: 1px solid green;
     height: 90%;
     width: 60%;
     margin: 10px;
     text-align: left;
-    background-color: RGBA(202,223,255,0.5);
+    background-color: RGBA(202,223,255,0.7);
```
Comment (nsantiagoblair) [inline style.css:201]: this might be silly but im used to seing it as `rgba(202, 223, 255, 0.7)` . So capitalization and spacing...

**Review by nsantiagoblair — COMMENTED** [review]
Looks good! 🎉 
Only things i commented were more about how to make it look a bit tidier

File: index.html (hunk 1 above)
Comment (karimkefi) [reply index.html:8]: not sure why the space is there. Looks ok on my IDLE

**Pull request merged**